documentation/docs/liquidity-mining/upcoming/DAI_on_Bamboo.md

```diff
@@ -0,0 +1,43 @@
+## Bamboo Relay
+[Bamboo Relay](https://bamboorelay.com/) is a decentralized crypto exchange built upon 0x protocol. Users can trade, lend or borrow any ERC20 token trustlessly from their own wallet. [0x](https://0x.org/) sponsors this program with a goal to improve the liquidity for its Relayers.   
+
+## Timeline
+
+This liquidity mining program will be initially running for 12 weeks from early February 2020 with probability for extension. 
+
+## Rewards
+
+Liquidity minders can earn rewards based on a formula that takes how long you've been placing orders over time, the spreads and your order sized into consideration. You'll be able to check your earned rewards by minute, and your total earned balance in the "activity" section. 
```

Comment:
Liquidity miners can earn rewards based on a formula that takes into account duration and consistency of orders placed over time, spreads, and sizes.

Reply:
(change throughout)